Frontend/src/FacialRecognition/FacialRecognitionModal.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useRef } from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import Webcam from 'react-webcam';
+
+const FacialRecognitionModal = ({ onClose, onImageCapture }) => {
+    const webcamRef = useRef(null);
+    const [imgSrc, setImgSrc] = useState(null);
+    const [error, setError] = useState(null);
+
+    const capture = () => {
+    const imageSrc = webcamRef.current.getScreenshot();
+    if (imageSrc) {
+        setImgSrc(imageSrc);
+        setError("Image Taken Correctly, you can close the screen.");
+        onImageCapture(imageSrc);
+    } else {
+        setError("Error Taking Image, try again.");
+    }
+    };
```

Comment:
nit: format your code to use indenting so there are not two closing parenthesis with the same alignment